### DISCUSSION TITLE
ProFormDigit去除千分位

### DISCUSSION THREAD

**Discussion by WT-Road** [ant-design/pro-components#6666] [Q&A]
需要实现输入银行账号时间隔四位加空格功能
编辑时没问题
![image](https://user-images.githubusercontent.com/14371925/221517045-402543fa-1e4c-41f1-be60-449ba2ad0956.png)
只读时如何去除千分位
![image](https://user-images.githubusercontent.com/14371925/221517230-703b6176-e99c-4e5c-b322-ff9fa44824be.png)

**Comment by aspirantzhang** [comment]
这个有方案吗，遇到同样尴尬的问题。。。

**Comment by aspirantzhang** [comment]
看了源码 https://github.com/ant-design/pro-components/blob/1f0c04d6d873effa1c300a43253f4f4257e31d7b/packages/field/src/components/Digit/index.tsx#L49-L68
似乎最方便的方法就是用 formatter 再把逗号删除
```
formatter: (number) => {
  return number.replace(/,/g, '');
},
```
